Editor/EditorCore/EditorHandleUtility.cs

```diff
@@ -554,6 +565,9 @@ static bool GetPlaneFromPickedObject(Vector2 mousePosition, out Plane plane)
                         plane = new Plane(
                             go.transform.TransformDirection(hit.normal),
                             go.transform.TransformPoint(hit.point));
+                        var forward = go.transform.forward;
+                        var dot = Vector3.Dot(plane.normal, forward);
+                        bitangent = Mathf.Abs(dot) > .9f ? go.transform.up : forward;
```

Comment:
this can result in a bitangent that is not parallel to the surface of the hit face. you'll need to do something like 

```
bitangent = nrm * (nrm *( |dot| > .9f ? transform.up : transform.forward ))
```

(where * is cross product, |x| is abs)